section1/exercises/variables.js

```diff
@@ -23,33 +23,56 @@ console.log(enrolledStudents);
 // YOU DO:
 // Write code below to save the string 'Harry Potter must not return to Hogwarts!'
 // log that variable to the console.
-
+var harryFate = 'Harry Potter must not return to Hogwarts!';
+console.log(harryFate);
 
 // YOU DO:
-// Declare three variables, named `firstName`, `isHungry` and `numberOfPets`. 
+// Declare three variables, named `firstName`, `isHungry` and `numberOfPets`.
   // Store the appropriate data types in each.
 // log all three variables to the console.
 
+var firstName = 'Nina';
+var isHungry = true;
+var numberOfPets = 20;
+console.log(firstName, isHungry, numberOfPets);
```

Comment:
✅ Awesome work printing all in one line. Make sure you know how to print on multiple lines as well :)

Reply:
console.log('' ,firstName, '\n' ,isHungry, '\n' ,numberOfPets);

@priyapower